dependencies/auth.py

```
@@ -0,0 +1,25 @@
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/auth/login")
+
+def get_current_user(token: str = Depends(oauth2_scheme)):
+    try:
+        payload = jwt.decode(
+            token,
+            jwt_settings.SECRET_KEY,
+            algorithms=[jwt_settings.ALGORITHM],
+        )
+    except JWTError:
+        raise HTTPException(status_code=401, detail="토큰 오류")
```

Ruff (0.14.11)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)